openy_traction_rec.info.yml

```diff
@@ -1,8 +1,8 @@
-name: Open Y Traction Rec
+name: YMCA Website Services Traction Rec
 type: module
 description: 'Provides Traction Rec integration.'
-core_version_requirement: ^9.5 || ^10
-package: 'YMCA Website Services'
+package: YMCA Website Services
+core_version_requirement: ^8.7.7 || ^9 || ^10
 configure: openy_traction_rec.settings
 dependencies:
   - key:key
```

Comment:
We need to improve composer.json with all these modules and limit versions, expecially for migrate and drush

Reply:
composer has this already:
```
        "drush/drush": "^12 || ^10",
        "drupal/migrate_plus": "^5.2 || ^6",
        "drupal/migrate_tools": "^5.1 || ^6",
```
What's missing?